docs/support/unbricking/unbrick-ch341a.md

```diff
@@ -100,13 +100,16 @@ You don't need to do this if flashing a stock firmware backup created by the Fir
 
 1. For both the options below, we'll need to use the cbfstool (coreboot filesystem) binary, so let's download/extract that:
     * `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz`
-    * Option 1: Extract VPD from the firmware on device
+    * Option 1: Extract VPD and HWID from the firmware on device
       * `sudo flashrom -p ch341a_spi -r badflash.rom`
       * `./cbfstool badflash.rom read -r RO_VPD -f vpd.bin`
-    * Option 2: Extract VPD from stock firmware backup created by Firmware Utility Script (this assumes the file has been copied into working directory)
+      * `./cbfstool backup.rom extract -n hwid -f hwid.txt`
+    * Option 2: Extract VPD and HWID from stock firmware backup created by Firmware Utility Script (this assumes the file has been copied into working directory)
       * `./cbfstool stock-firmware-<devicename>-<date>.rom read -r RO_VPD -f vpd.bin`
-      * Then we inject the VPD into the firmware image to be flashed.
+      * `./cbfstool stock-firmware-<devicename>-<date>.rom extract -n hwid -f hwid.txt`
```

Comment:
This won't work, the HWID is not stored as a text file in CBFS in the stock firmware.
You need to use gbb_utility to read it from the stock firmware, just like my script does.